modules/edu.gemini.seqexec.server/src/main/java/edu/gemini/seqexec/server/gcal/BinaryOnOff.java

```diff
@@ -0,0 +1,9 @@
+package edu.gemini.seqexec.server.gcal;
```

Comment:
Yes. I have a call to a java class method (CaStatusServer.addEnum) that requires an java Enum class as a parameter.